src/main/java/com/github/dbmdz/solrocr/util/SolrVersionInfo.java

```diff
@@ -0,0 +1,32 @@
+package com.github.dbmdz.solrocr.util;
+
+import org.apache.lucene.util.Version;
+import org.apache.solr.client.api.util.SolrVersion;
+
+public class SolrVersionInfo {
```

Comment:
Should we rename this class to `VersionInfo` because it handles both the Lucene and Solr versions?

Reply:
We absolutely should, thanks :-)